.github/workflows/e2e-node.yml

```diff
@@ -0,0 +1,39 @@
+name: End-to-end node tests
+
+on:
+  push:
+
+env:
+  CI: true
+jobs:
+  e2e-node:
```

Comment:
Note: we need to make sure we update the branch protection rules with this change.